databases/week2/week2_part3.sql

```diff
@@ -0,0 +1,41 @@
+/* 
```

Comment:
you should use data for lesson2 for this part.
https://github.com/HackYourFuture-CPH/databases/blob/master/lesson2/lesson2-data.sql